WallProject/WallProject/Models/DTO/CategoryDTO.cs

```diff
@@ -7,7 +7,7 @@ namespace WallProject.Models.DTO
 {
     public class CategoryDTO
     {
-        public int id { get; set; }
+        public int categoryId { get; set; }
```

Comment:
Wydaje mi się że 'id' było zgodne z dokumentacją i nie wymaga zmiany

Reply:
Albo dokumentacja albo to co przesyłamy w endpointach. Jedno z tych 2 jest źle